recipe/build.sh

```diff
@@ -74,6 +128,7 @@ if [ $(uname) == Darwin ]; then
                 -arch $(uname -m) \
                 -platform unsupported/macx-clang-libc++ \
                 -silent \
+                -no-dbus \
                 -sdk $(xcode-select --print-path)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk \
     ####
 
```

Comment:
Now that linux passed you could set `make -j $CPU_COUNT`

Reply:
I will, thanks!
just waiting windows to finish